deploy_zkevm_contracts.star

```diff
@@ -125,3 +129,18 @@ def run(plan, args):
         service_name="contracts" + args["deployment_suffix"],
         src="/opt/zkevm/dynamic-kurtosis-allocs.json",
     )
+
+    # Force update GER
```

Comment:
Can you document why we need to update the global exit root here? I know it's related to a bug in the first batch of cdk-erigon but it would be nice to document it here so we can get rid of it once the fix is ready :)

Reply:
sure, where do you think I should write about this?

Reply:
Here is fine. The closer to the code the better :)

Maybe also add a `TODO: Remove this hack once the cdk-erigon bug is resolved` so that we can find it quickly.

Reply:
this is not a bug, but a feature. The contract doesn't allow for this to happen. IMO it should, but since there are no plans for that, this is a feature 🙃